File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/save-person-unit-library-info_param.ts


import { ISavePersonUnitLibraryInfo_Param } from './isave-person-unit-library-info_param';

export class SavePersonUnitLibraryInfo_Param implements ISavePersonUnitLibraryInfo_Param {
  
	public PersonsUnitLibraryInfoID?: number;
	public PersonID?: number;
	public UnitID?: number;
	public JobTitle: string = "";
	public YearsInPosition?: number;
	public WorkEmailAddress: string = "";
	public RankID?: number;
	public IsUnitCommander?: boolean;
	public PoliceMilSecID: string = "";
	public HostNationPOCName: string = "";
	public HostNationPOCEmail: string = "";
	public IsVettingReq?: boolean;
	public IsLeahyVettingReq?: boolean;
	public IsArmedForces?: boolean;
	public IsLawEnforcement?: boolean;
	public IsSecurityIntelligence?: boolean;
	public IsValidated?: boolean;
  
}

File: GTTS/INL.PersonService.Models/SavePersonUnitLibraryInfo_Param.cs

namespace INL.PersonService.Models
{
    public class SavePersonUnitLibraryInfo_Param : ISavePersonUnitLibraryInfo_Param
    {
        public long? PersonsUnitLibraryInfoID { get; set; }
        public long? PersonID { get; set; }
        public long? UnitID { get; set; }
        public string JobTitle { get; set; }
        public int? YearsInPosition { get; set; }
        public string WorkEmailAddress { get; set; }
        public int? RankID { get; set; }
        public bool? IsUnitCommander { get; set; }
        public string PoliceMilSecID { get; set; }
        public string HostNationPOCName { get; set; }
        public string HostNationPOCEmail { get; set; }
        public bool? IsVettingReq { get; set; }
        public bool? IsLeahyVettingReq { get; set; }
        public bool? IsArmedForces { get; set; }
        public bool? IsLawEnforcement { get; set; }
        public bool? IsSecurityIntelligence { get; set; }
        public bool? IsValidated { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}
